shared/vehicles.lua

```diff
@@ -5339,7 +5339,7 @@ QBShared.Vehicles = {
         ['shop'] = 'truck',
     },
     ['mixer'] = {
-        ['name'] = 'Cement Mixer',
+        ['name'] = 'HVY Cement Mixer',
```

Comment:
```suggestion
        ['name'] = 'Cement Mixer',
```

The brand is not needed in the actual label of the vehicle.